.github/workflows/Image-build-Egov-Filestore.yaml

```
@@ -0,0 +1,89 @@
+          format: 'SARIF'
+      - name: Upload SARIF result to GitHub
+        uses: github/codeql-action/upload-sarif@v3
+        with:
+          sarif_file: reports/dependency-check-report.sarif
+        if: always()
+
+      # Login to Azure Container Registry
+      - name: Login to Azure Container Registry
+        run: docker login -u pucarsolutionsdev -p ${{ secrets.ACR_PASSWORD }} pucarsolutionsdev.azurecr.io
+      - name: Git commit_hash for service
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:2:36: Double quote to prevent globbing and word splitting

(shellcheck)